src/d3a/d3a_core/sim_results/area_throughput_stats.py

```diff
@@ -34,17 +34,18 @@ def _calc_results(self, area, baseline_value, energy_profile, direction_key):
         # the numbers are rounded for both local and redis results
         peak_percentage = \
             round_floats_for_ui(max(energy_profile.values(), default=0.0) / baseline_value * 100)
-        peak_energy_kWh = round_floats_for_ui(max(energy_profile.values(), default=0.0))
         if direction_key == "import":
             baseline_peak_energy_kWh = round_floats_for_ui(area.baseline_peak_energy_import_kWh)
         else:
             baseline_peak_energy_kWh = round_floats_for_ui(area.baseline_peak_energy_export_kWh)
         return {
-            "peak_energy_kWh": peak_energy_kWh,
             "peak_percentage": peak_percentage,
             "baseline_peak_energy_kWh": baseline_peak_energy_kWh,
          }
 
+    def _calc_peak_energy_results(self, energy_profile):
+        return {"peak_energy_kWh": round_floats_for_ui(max(energy_profile.values(), default=0.0))}
+
```

Comment:
This is just refactoring, right?

Reply:
No, this is actually a requirement from the bug ticket. Sarah wants the `peak_energy_kWh` value to always be reported for every area.

Reply:
Ah yes, makes sense! This is the value the user has to see to set a Baseline to see the percentage in the next run. Thanks!